### ISSUE TITLE
Generating Without Scene Conditioning Images & Datasets Used for Training

### ISSUE THREAD

**Issue by eliasShah** [Stability-AI/stable-virtual-camera#45]
Hello,

First, thank you for this amazing work!

I have two questions:

**1. Conditioning Input:**
I understand that conditioning on at least one scene image is important for the model. However, is it possible to generate views without any conditioning image (i.e., setting P=0 in the P+Q sampling regime mentioned in the paper)? I experimented with the CLI demo and created a train_test_split file (similar to the ones provided in the [assets_demo_cli](https://github.com/Stability-AI/stable-virtual-camera/releases/tag/assets_demo_cli)) that contains no input image, but the code did not accept this configuration.

**2. Training Dataset:**
Could you clarify whether the model was trained on all the datasets mentioned in the Benchmark section of the paper? 
<img width="448" alt="Image" src="https://github.com/user-attachments/assets/27a564e3-c9d9-46b1-8e77-4fe36d97fdad" />

Thanks,
Elias

**Comment by XinyaChen21** [comment]
Hello! I also have questions regarding the Training Dataset. I noticed that some datasets have different evaluation splits, e.g. RealEstate10K, CO3D, WildRGB-D, DL3DV-140, Tanks and Temples. Are these splits all subsets of the original test set? Was only the original training set used during training?

**Comment by MinJunKang** [comment]
![Image](https://github.com/user-attachments/assets/f57f0f80-02f6-4077-a68d-06ef57f960b4)
In the paper, it seems the model is trained on multiple datasets (all the datasets maybe) following the comment.

**Comment by jensenz-sai** [comment]
Hi @eliasShah @XinyaChen21 @MinJunKang,

- Sadly, no, as the model is only trained with at least one input image.
- Unfortunately, we can't disclose the exact training datasets we use for the training, but it is a pool of different datasets covering both object-level and scene-level datasets. @XinyaChen21 All test scenes are excluded from the training data. Most of the datasets in the benchmark sections are out-of-distribution. 

Hope it helps!

**Comment by XinyaChen21** [comment]
Thank you for your reply! You mentioned that “most of the datasets in the benchmark sections are out-of-distribution.” Does this imply that certain datasets included in the benchmark had training splits that were not used during training?
